packages/server/src/model/database-layer.ts

```diff
@@ -292,6 +292,15 @@ export const spec = {
     response: t.void,
     canBeNull: false,
   },
+  TaxonomyCreateEntityLinkMutation: {
```

Comment:
```suggestion
  TaxonomyCreateEntityLinksMutation: {
```

Can you change the mutation name in db-layer as well. Sorry for not thinking about the perfect name before... :see_no_evil: